utility/SSD_writeback_cache.py

```diff
@@ -0,0 +1,64 @@
+import json
+import logging
+import os
+import random
+import subprocess
+import sys
+import time
+
+log = logging.getLogger(__name__)
+
+logging.basicConfig(
+    handlers=[logging.StreamHandler(sys.stdout)],
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(filename)s:%(lineno)d - %(message)s",
+)
+
+
+def ssd_write_cache():
+    """
+
+    This method to validate constant flushing and
+    evicting of data for an extended period of time.
+    This scenario in particular
+    recommend writing a script that would run through a few thousand
+    iterations of SIGKILLed "rbd bench" "fio" on the same image:
+
+    """
+    size = 1 * 1024 * 3
+    pool_name = "ssd"
+    image_name = "image3"
+    os.system(f"ceph osd pool create {pool_name} 128")
+    os.system(f"ceph osd pool application enable {pool_name} rbd")
+    os.system(f"rbd pool init -p {pool_name}")
+    os.system(f"rbd create --size {size} {pool_name}/{image_name} --thick-provision")
+    for i in range(1, 10000):
+        print("#" * 80)
+        print(f"Iteration {i}")
+        print("#" * 80)
+        proc = subprocess.Popen(
+            [
+                f"fio --name=test-1 --ioengine=rbd --pool={pool_name} "
+                f"--rbdname={image_name} --numjobs=2 --rw=write --bs=4k "
+                f"--iodepth=8 --fsync=32 --runtime=80 --time_based "
+                f"--group_reporting --ramp_time=120"
+            ],
+            shell=True,
+        )
+        timeout = random.randint(45, 100)
```

Comment:
IMO, we should check the storage capacity before going to the next iteration. We could break the cluster if it is not properly sized.

Reply:
As discussed, will consider this storage capacity as improvements and enhancement in next sprint